srcpkgs/timeshift/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'timeshift'
 pkgname=timeshift
 version=19.01
-revision=1
+revision=2
```

Comment:
no need to revbump when fixing broken build

Reply:
Done